src/axom/sidre/examples/sidre_data_vs_metadata.cpp

```diff
@@ -60,12 +60,17 @@ int main(int argc, char* argv[])
   // group has one view and the datastore has one buffer (holding the
   // view's data).
   //
-  // Initialize the elements of the integer array.  Then, deallocate the view.
+  // Initialize the elements of the integer array.
   //
-  // The view and its description of the data remains, but the buffer
-  // holding the integer array is destroyed on de-alloction since there
+  // Then, deallocate the view. The view and its description of the data
+  // remains, but the buffer holding the array is deallocted since there
   // is only one view referencing its data.
```

Comment:
```suggestion
  // was only one view referencing its data.
```
I like the past tense because there is no longer a buffer: you're describing a relationship that existed before the operation and no longer exists.

Reply:
Got it.